Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.1.2-alpine
+
+ENV HOME=/usr/rss-watcher
+
+RUN mkdir -p $HOME
+
+WORKDIR $HOME
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh 
```

Comment:
Just want to note that this is pretty important for other repos that adopt a similar setup. Without this, we can't pull in the `service-util` code.